website/content/gateway/api_reference/resources/authorizations/series/series.md

```diff
@@ -18,7 +18,7 @@ Default: `recurring`.
 {{% regex_optional %}}Conditional. Cannot be present if `series[previous][id]` or any `series[previous][visa][...]` is present. {{% /regex_optional %}}{{% /description_details %}}
 
 
-{{% description_term %}}series[previous][mastercard][tid] {{% regex %}}[A-Za-z0-9]{3}[A-Za-z0-9]{6}[0-9]{4} {2}{{% /regex %}}{{% /description_term %}}
+{{% description_term %}}series[previous][mastercard][tid] {{% regex %}}[A-Za-z0-9]{3}[A-Za-z0-9]{6}[0-9]{4}  {{% /regex %}}{{% /description_term %}}
```

Comment:
I think the previous with ` {2}` is more clear on a deliberate intent of having two spaces in the end 🤷